components/activity_and_insights/insights/insights_header/insights_header.scss

```diff
@@ -0,0 +1,35 @@
+.Insights___header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 14px 12px;
+    border-bottom: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
+    background-color: var(--center-channel-bg);
+    grid-area: header;
+
+    h2 {
```

Comment:
I think we should consider making this dropdown a button instead of just an h2 tag - just like it is in the channel headaer